src/main/resources/assets/nvidium/lang/en_us.json

```diff
@@ -1,26 +1,26 @@
-  "nvidium.options.translucency_sorting.quads" : "Quads",
-  "nvidium.options.statistics_level.name" : "Statistics Level",
+  "nvidium.options.pages.nvidium": "Nvidium",
+  "nvidium.options.region_keep_distance.name": "Region Keep Distance",
+  "nvidium.options.region_keep_distance.tooltip": "Distance to keep loaded regions",
+  "nvidium.options.automatic_memory_limit.name": "Automatic memory limit",
+  "nvidium.options.automatic_memory_limit.tooltip": "Automatically determines the memory limit to set given the amount of available VRAM on your system (Close and reopen settings to edit the max memory)",
+  "nvidium.options.max_gpu_memory.name": "Max GPU memory",
+  "nvidium.options.max_gpu_memory.tooltip": "Max GPU memory allowed, will start to cull chunks if this limit is hit",
+  "nvidium.options.enable_temporal_coherence.name": "Enables temporal coherence",
+  "nvidium.options.enable_temporal_coherence.tooltip": "Removes artifacting when turning around",
+  "nvidium.options.mb": "%s MB",
+  "nvidium.options.translucency_sorting.name": "Translucency Sorting",
+  "nvidium.options.translucency_sorting.tooltip": "Translucency sorting level, each level has different performance impact and visual quality. \nNone:No translucency sorting, no impact, can look quite bad\nSections: Section level translucency sorting, brings translucency to the same level as Sodium, minimal impact\nQuads: Incremental sorting, sorts geometry correctly over multiple frames, can cause visual weirdness while sorting",
+  "nvidium.options.translucency_sorting.none": "None",
+  "nvidium.options.translucency_sorting.sections": "Sections",
+  "nvidium.options.translucency_sorting.quads": "Quads",
+  "nvidium.options.statistics_level.name": "Statistics Level",
   "nvidium.options.statistics_level.tooltip" : "Statistics logging level, tracks the visibility count of cull layers",
-  "nvidium.options.statistics_level.none" : "None",
-  "nvidium.options.statistics_level.frustum" : "Frustum",
-  "nvidium.options.statistics_level.regions" : "Regions",
-  "nvidium.options.statistics_level.sections" : "Sections",
-  "nvidium.options.statistics_level.quads" : "Quads",
-  "nvidium.options.async_bfs.name" : "Enable async bfs",
-  "nvidium.options.async_bfs.tooltip" : "Enables asynchronous bfs chunk section loading, greatly reduces the frame time when moving, more noticeable with higher render distances"
-}
\ No newline at end of file
+  "nvidium.options.statistics_level.none": "None",
+  "nvidium.options.statistics_level.frustum": "Frustum",
+  "nvidium.options.statistics_level.regions": "Regions",
+  "nvidium.options.statistics_level.sections": "Sections",
+  "nvidium.options.statistics_level.quads": "Quads",
+  "nvidium.options.async_bfs.name": "Enable ASync BFS",
+  "nvidium.options.async_bfs.tooltip": "Enables asynchronous BFS chunk section loading, greatly reduces the frame time when moving, more noticeable with higher render distances"
+}
```

Comment:
Although I agree that there shouldn't be a space before the colons, this isn't a formatting PR.